bower.json

```
@@ -1,7 +1,7 @@
 {
     "name": "adapt-background-switcher",
     "repository": "git://github.com/cgkineo/adapt-background-switcher",
-    "framework": "^2.0.0",
+    "framework": ">=2.0.0",
```

```suggestion
    "framework": ">=2",
```
not sure how much actual difference that makes but that's what all the others are set to so let's be consistent since it's a small change